accumulator/utils.go

```diff
@@ -10,6 +10,15 @@ var verbose = false
 
 // takes a slice of dels, removes the twins (in place) and returns a slice
 // of parents of twins
+//
+// Example with 15-leaf tree in printout.txt line 21:
+// If deleting the nodes 4 through 9, the function would be called with
```

Comment:
shouldn't it be 4 through 8?

Reply:
I changed it to be 4 though 9 everywhere to be consistent with the other PR.

Reply:
That works but if you are deleting leaves 4-9, then 8 has a sibling (9) so the below comment would be off where it states:

```only 8 is not a "twin" (it has no siblings to be deleted, note that 7 is in a different tree)```

Above you mention that 20 is returned as a parent. 20 would not be returned if 8 didn't have a sibling.

For deletion of nodes [4, 5, 6, 7, 8, 9], the returning values would be ```parents = [18, 19, 20]``` and ```dels=[]```